javascript-loops-practice/odd-or-even.js

```diff
@@ -0,0 +1,17 @@
+/* exported oddOrEven
+a list of numbers
+needs to sort through list and find even or odd numbers.
+if the number is even change the number to 'even'
+if the number is odd change the number to 'odd'
+*/
+function oddOrEven(numbers) {
+  for(var i = 0; i < numbers.length; i++) {
+    if(numbers[i] % 2 === 0) {
+      numbers[i] = 'even';
+    }
+    else if(numbers[i] % 2 !== 0) {
+      numbers[i] = 'odd';
+    }
+  }
```

Comment:
This is **_REALLY_** bad. You are changing the original array. If anybody calls this function with an array of numbers, all the numbers will be gone. You should be making a new array holding the proper values to return.